src/Navigation.ts

```diff
@@ -1,22 +1,37 @@
 import { activePlugins } from './Utils/constants';
 
+const getAllExtensions = async () => {
+  return (
+    await Promise.all(
+      activePlugins.flatMap(async (pluginName) => {
+        const { extensions } = (await (await fetch(`/api/plugins/${pluginName}/plugin-manifest.json`))?.json()) || {};
+        return extensions;
+      }),
+    )
+  ).flat();
+};
+
+const claculateRoutes = ([appId, navSection], currentNamespace, extensions) => {
+  return extensions
+    .filter(({ type, properties }) => type.includes('console.navigation/href') && properties.section === navSection)
```

Comment:
What about the extensions that do not have a `properties.section`, is `navSection` just not defined and `undefined === undefined`?

Reply:
If the `properties.section` is not defined the `navSection` will not be defined as well. So this condition would choose such nav item. There's a reason why I would like to bring third param to the `dynamicNav`, everything behind `~` would be matched agains for instance dynamic plugin name. We could send whichever information from the navigation builder we choose to.